pkg/cmd/repo/credits/credits.go

```diff
@@ -161,7 +162,7 @@ func creditsRun(opts *CreditsOptions) error {
 
 	logins := []string{}
 	for x, c := range result {
-		if isTTY && !static {
+		if c.Type == "User" && isTTY && !static {
```

Comment:
This only skips bots in the default (animated) view, but not in the `-s` mode nor when the output is redirected.

Since this `if` clause is for the purpose of rendering styles, I feel that we should add the code to skip bots somewhere outside of it, earlier in the code. How about something like this?

```go
for x, c := range result {
  if c.Type != "User" {
    continue
  }
  // ...
}
```

Reply:
This works for me. I did not catch this in the non-animated mode but can confirm it was missed.

Reply:
I updated the code to leverage your recommendation. Having the check is explicit and clear enough for me.